### PR TITLE
GUI Support Using Tkinter

### LINKED ISSUE #16: Make a GUI for the project using Tkinter
**Actual Behaviour**
* The application is only CLI based and doesn't have a GUI as of now.

**Expected Behaviour**
* Make a GUI for the project using Tkinter

### PR DESCRIPTION
**Pull request by jains8844**
Fixes #16 

Changes: Addition of a file : app.py for tkinter support. 
Now the application for it will be run by the command `python app.py` in the terminal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,6 @@ It is implemented using the **BeautifulSoup** module of **bs4** , **requests** m
 
 **Instructions before using:** If you are going to use the third feature, make sure that all the folders' name contain only the movie name, and no other extra character or whitespace should be left. Also ensure you enter the correct and complete path (e.g.: _/home/user-name/Videos/_  or _/media/user-name/New Volume/Videos_).
 
-**To run the script:** Type `python imdb.py` from your terminal.
+**To run the script:** Type `python app.py` from your terminal.
```
Comment (akshatnitd) [inline README.md]: Do not replace the CLI method.
Instead add a new line to run GUI

**Comment by jains8844** [comment]
Should I copy the code for GUI Separately ?

**Comment by jains8844** [comment]
Kindly check now

**Comment by jains8844** [comment]
Done

**Comment by jains8844** [comment]
The requirements.txt file has been changed since I added requirement for python-tk(Tkinter).

**Comment by jains8844** [comment]
I reverted the change and now the branch has no conflict. Attached are the screenshots for the gui.
screenshot from 2018-12-06 21-52-09
screenshot from 2018-12-06 21-54-03
screenshot from 2018-12-06 21-54-46
screenshot from 2018-12-06 21-55-56
screenshot from 2018-12-06 21-56-02

**Comment by jains8844** [comment]
I added the instructions for installint tkinter.
But it shows conflict with that.
Kindly check.
Resolved other conflicts.

**Comment by akshatnitd** [comment]
Go through this and try resolving the conflict.
https://help.github.com/articles/resolving-a-merge-conflict-on-github/

**Comment by jains8844** [comment]
Done

File: README.md [hunk 2]
```diff
@@ -18,13 +18,19 @@
...
+   To install Tkinter on Python 3.x  : `sudo apt-get install python3-tk`
+
 
 It is implemented using the **BeautifulSoup** module of **bs4** , **requests** module, along with **os**, **sys** and **json** modules.
-* The first feature is implemented using the json response generated, after making a request via the **requests** module to the [TMDB API](https://www.themoviedb.org/) after generating an API key from the website. The json response is thereafter processed and the information is extracted using the **json** module. The IMDB rating is obtained by using the IMDB ID obtained from the json response and then using the ID to scrap rating from : http://www.imdb.com/title/<IMDB_ID> .
+* The first feature is implemented using the json response generated, after making a request via the **requests** module to the [IMDB API](https://www.themoviedb.org/) after generating an API key from the website. The json response is thereafter processed and the information is extracted using the **json** module. The IMDB rating is obtained by using the IMDB ID obtained from the json response and then using the ID to scrap rating from : http://www.imdb.com/title/<IMDB_ID> .
```
Comment (akshatnitd) [inline README.md]: It should be `TMDB`. The change not required.

**Comment by jains8844** [comment]
Sure 👍

**Pull request merged**